pyspline/pyVolume.py

```diff
@@ -251,9 +253,8 @@ def __init__(self, recompute=True, **kwargs):
                     self.calcParameterization()
                 else:
                     Error(
```

Comment:
Can we put a more specific error here, such as KeyError? This comes up throughout the code, in regions unaffected by this PR, so we can save this for later as well.

Reply:
This is a separate issue and something on the back of my mind. I think the pros/cons are a bit unclear at the moment, let's discuss this with the lab some time.